site/docs/v1/tech/admin-guide/upgrade.adoc

```diff
@@ -7,41 +7,270 @@ navcategory: admin
 * <<FastPath>>
 * <<Database>>
-* <<Themes>>
-* <<Downtime>>
 
-If you used the FastPath or other installation methods initially, you may prefer to use that same mode to download the latest version of FusionAuth. If you would like to continue by manually downloading the necessary packages, read on.
+== When to Upgrade
+
+=== Reasons to Upgrade
+
+Upgrading to a newer version of FusionAuth can provide the following benefits:
+
+* Access to new features and improvements.
+* Improved performance and stability.
+* Critical Security updates and patches.
+
+=== Reasons not to Upgrade
+
+You might want to delay upgrading in the following situations:
+
+* Your current version meets all your requirements.
+* You are in the middle of a project or have tight deadlines.
+* You have concerns about compatibility with customizations or third-party integrations.
+
+=== Being Aware of Upgrade Options
+
+You can stay informed about new FusionAuth releases and updates by:
+
+* Monitoring the link:/docs/v1/tech/release-notes[release notes].
+* Joining the https://fusionauth.io/community[FusionAuth Community site] and https://fusionauth.slack.com/join/shared_invite/zt-6ykfov9u-MTwA1A6ptv0Gr6KcFBVG4w#/shared-invite/email[Slack channel].
+* Following FusionAuth on social media platforms, including https://twitter.com/FusionAuth[Twitter] and https://www.linkedin.com/company/fusionauth[LinkedIn].
+* Monitoring the https://github.com/FusionAuth/[FusionAuth GitHub repository].
+
+== How to Think About Upgrades
+
+Before upgrading FusionAuth, there are a number of things to consider. Go through each of the points below to see if any of them apply to your situation.
+
+=== Update Contents and Notes
+
+Read the link:/docs/v1/tech/release-notes[release notes]. Release notes help you understand how an update affects you. They include details on new features, improvements, bug fixes, and potential breaking changes.
+
+=== Testing
+
+Before starting the upgrade process on production, test the upgrade in a staging or development environment to identify potential issues. This is particularly important on more complex upgrades involving multiple version jumps or if you have made customizations to FusionAuth.
+
+=== Downtime and Database Migrations
+
+Plan for any downtime or maintenance windows required for the upgrade. If changes to the database schema are required, this could require more steps and downtime that affects end users. A pure application upgrade, on the other hand, may only affect admin functionality and not require any downtime from an end-user perspective.
+
+FusionAuth can perform database migrations automatically and silently using a process called link:/docs/v1/tech/guides/silent-mode[Silent Mode]. 
+
+If you want to run the migrations interactively, use link:/docs/v1/tech/installation-guide/fusionauth-app#maintenance-mode[Maintenance Mode], or you can run any needed scripts out of band (meaning directly on the database yourself).
```

Comment:
maintenance mode is never used when the runtime mode is production. they could run the migrations by:

* removing all server nodes from a load balancer/proxy
* running the db scripts
* adding all server nodes back to the load balancer/proxy

Reply:
is this resolved @sixhobbits ?